src/features/readme.md

```
@@ -0,0 +1,17 @@
+## Feature config overrides
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)